packages/config/tests/api/tests.mjs

```diff
@@ -1,8 +1,6 @@
-'use strict'
```

Comment:
Some test files must be `.mjs` due to `ava` not working well in monorepos mixing CommonJS and ESM packages. Once all packages of this monorepos are switched to ESM, those files can be renamed to `.js` and this comment can be removed.